content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -31,11 +31,11 @@ The Istio data plane components, the Envoy proxies, handle data flowing through
 the system. The Istio control plane component, Istiod, configures
 the data plane. The data plane and control plane have distinct performance concerns.
 
-## Performance summary for Istio {{< istio_release_name >}}
+## Performance summary for Istio 1.19
```

Comment:
Just verifying that the vCPU and memory numbers didn't seem to change at all.

Reply:
These numbers are mostly made up IMO, we haven't changed since Istio 1.0 lol...

Reply:
Can we remove them then? It's better not to say anything if we can't validate it.